Applications/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 add_subdirectory(MainApplication)
+add_subdirectory(MainApplication_BE)
```

Comment:
Please remove this subproject from your PR, since it is not related to the refactoring.